cargo-shuttle/src/lib.rs

```diff
@@ -1632,14 +1628,16 @@ impl Shuttle {
         raw: bool,
     ) -> Result<()> {
         let client = self.client.as_ref().unwrap();
-        if self.track_deployment_status(proj_id, depl_id).await? {
+        let failed = self.track_deployment_status(proj_id, depl_id).await?;
+        if failed {
             for log in client.get_deployment_logs(proj_id, depl_id).await?.logs {
                 if raw {
                     println!("{}", log.line);
                 } else {
                     println!("{log}");
                 }
             }
+            return Err(anyhow!("Deployment failed"));
```

Comment:
Should we add suggested actions for debugging the failure, or will that be apparent from the printed logs?